lib/ansible/modules/cloud/amazon/elb_classic_lb_facts.py

```diff
@@ -38,6 +38,9 @@
 extends_documentation_fragment:
     - aws
     - ec2
+requirements:
+  - botocore
+  - boto3
 '''
 
 EXAMPLES = '''
```

Comment:
`RETURNS` docs would be super cool here as well.

Reply:
Addressed.